### PR TITLE
Implement "getNodesUrls" standard

### PR DESCRIPTION
**Pull request by Elli610**

### CODE REVIEW

File: wallet/dapps-communication.md [hunk 1]
```diff
@@ -417,6 +417,58 @@ required:
 </tbody>
 </table>
 
+#### Get nodes url
```
Comment (gregLibert) [inline wallet/dapps-communication.md:420]:
```suggestion
#### Node URLs
```

File: wallet/dapps-communication.md [hunk 2]
```diff
@@ -417,6 +417,58 @@ required:
...
+
+This method is used to get the nodes url from the extension.
```
Comment (gregLibert) [inline wallet/dapps-communication.md:422]:
```suggestion
This method is used to get the node URLs known by the extension.
```

File: wallet/dapps-communication.md [hunk 3]
```diff
@@ -417,6 +417,58 @@ required:
...
+<th>Direction</th>
+<th>Type</th>
+<th>Format</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Webpage to extension</td>
+<td><code>Provider.getNodesUrl</code></td>
```
Comment (gregLibert) [inline wallet/dapps-communication.md:436]:
```suggestion
<td><code>Provider.getNodeUrls</code></td>
```

File: wallet/dapps-communication.md [hunk 4]
```diff
@@ -417,6 +417,58 @@ required:
...
+<td>none</td>
+<td><code>null</code></td>
+</tr>
+<tr>
+<td>Extension to webpage</td>
+<td><code>Provider.getNodesUrl.response</code></td>
```
Comment (gregLibert) [inline wallet/dapps-communication.md:442]:
```suggestion
<td><code>Provider.getNodeUrls.response</code></td>
```

File: wallet/dapps-communication.md [hunk 5]
```diff
@@ -417,6 +417,58 @@ required:
...
+
+```yaml
+type: object
+properties:
+  response:
+    type: string array
+  message:
+    type: string
+required:
+  - nodesUrl
```
Comment (gregLibert) [inline wallet/dapps-communication.md:453]:
```suggestion
type: array
items:  
    type: string
    format: URL
```

File: wallet/dapps-communication.md [hunk 6]
```diff
@@ -417,6 +417,58 @@ required:
...
+
+</td><td>
+  
+```json
+{
+   "nodesUrl":  [
+                  "http://localhost:1234", 
+                  "https://massa-nodes.net"
+                ]
+}
```
Comment (gregLibert) [inline wallet/dapps-communication.md:464]:
```suggestion
[
  "http://localhost:1234", 
  "https://massa-nodes.net"
]
```

File: wallet/dapps-communication.md [hunk 7]
```diff
@@ -465,4 +517,7 @@ const signature = await myAccount.sign([0, 1, 2]);
 
 // Delete an account.
 await myProvider.importAccount(myAccount.address());
+
+// Get nodes url
+await myProvider.getNodesUrl();
```
Comment (gregLibert) [inline wallet/dapps-communication.md:522]:
```suggestion
const urls = await myProvider.getNodeUrls();
```

**Review by gregLibert — CHANGES_REQUESTED** [review]

**Review by BatiGencho — APPROVED** [review]

**Review by gregLibert — CHANGES_REQUESTED** [review]
Can you update the version of the standard to 0.2 ?

**Comment by Elli610** [comment]
> Can you update the version of the standard to 0.2 ?

Done

**Review by gregLibert — APPROVED** [review]

**Pull request merged**